### PR TITLE
BO - Brands & Suppliers - the red alert "Image format not recognized" is missing the webp

### LINKED ISSUE #28986: BO - Brands & Suppliers - the red alert "Image format not recognized" is missing the webp
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

After merging this PR: https://github.com/PrestaShop/PrestaShop/pull/24394, 8.0.0 is compatible with webp
In the BO > Catalog > Brands & Suppliers page, the red alert "Image format not recognized" is missing the new type webp
image

### Expected behavior

It should be as in the categories page
image

### Steps to reproduce

1. Go to BO > Catalog > Brands page
2. Add new brand
3. Upload a logo with an invalid format (SVG)
4. Save
5. See error: the red alert is missing the webp type
6. Go to BO > Catalog > Suppliers page
7. Add new supplier
8. Upload a logo with an invalid format (SVG)
9. Save
10. See error: the red alert is missing the webp type

### PrestaShop version(s) where the bug happened

8.0.x

### PHP version(s) where the bug happened

7.2

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by ghost**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x
| Description?      | BO - Brands & Suppliers - the red alert "Image format not recognized" is missing the webp
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #28986.
| Related PRs       | 
| How to test?      | See issue
| Possible impacts? |

### CODE REVIEW

File: src/PrestaShopBundle/Controller/Admin/Sell/Catalog/ManufacturerController.php [hunk 1]
```diff
@@ -681,7 +681,7 @@ private function getErrorMessages()
                         $iniConfig->getUploadMaxSizeInBytes(),
                     ]),
                 UploadedImageConstraintException::UNRECOGNIZED_FORMAT => $this->trans(
-                    'Image format not recognized, allowed formats are: .gif, .jpg, .png',
+                    'Image format not recognized, allowed formats are: .gif, .jpg, .png .webp',
```
Comment (PrestaEdit) [inline src/PrestaShopBundle/Controller/Admin/Sell/Catalog/ManufacturerController.php]:
```suggestion
                    'Image format not recognized, allowed formats are: .gif, .jpg, .png, .webp',
```

File: src/PrestaShopBundle/Controller/Admin/Sell/Catalog/SupplierController.php [hunk 2]
```diff
@@ -502,7 +502,7 @@ private function getErrorMessages()
                         $iniConfig->getUploadMaxSizeInBytes(),
                     ]),
                 UploadedImageConstraintException::UNRECOGNIZED_FORMAT => $this->trans(
-                    'Image format not recognized, allowed formats are: .gif, .jpg, .png',
+                    'Image format not recognized, allowed formats are: .gif, .jpg, .png .webp',
```
Comment (PrestaEdit) [inline src/PrestaShopBundle/Controller/Admin/Sell/Catalog/SupplierController.php]:
```suggestion
                    'Image format not recognized, allowed formats are: .gif, .jpg, .png, .webp',
```

**Review by PrestaEdit — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Thank you @PrestaEdit

**Review by Hlavtox — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
Hello @okom3pom ,

In Brands and Suppliers, the error message is updated.
Screenshot 2022-07-19 at 10 07 05

It is QA ✅

**Comment by kpodemski** [comment]
Thank you @okom3pom @florine2623

**Pull request merged**